lib/src/react_client/synthetic_data_transfer.dart

```diff
@@ -0,0 +1,111 @@
+import 'dart:html';
+
+import 'package:react/src/react_client/synthetic_event_wrappers.dart' as events;
+
+/// Used to wrap `SyntheticMouseEvent.dataTransfer`'s interop value in order to avoid known errors.
+///
+/// See <https://github.com/cleandart/react-dart/pull/220> for more context.
+///
+/// Related: [syntheticDataTransferFactory]
+class SyntheticDataTransfer {
```

Comment:
This existed previously, correct? Was it just moved into its own file?

Reply:
Yep, exactly!